spring-rabbitmq-tuning-lib/src/main/java/com/tradeshift/amqp/autoconfigure/TunedRabbitAutoConfiguration.java

```diff
@@ -293,6 +296,18 @@ private void autoCreateQueues(TunedRabbitProperties properties, RabbitAdmin rabb
         }
     }
 
+    private boolean isAValidExchange(TunedRabbitProperties properties) {
+        return DIRECT.equals(properties.getExchangeType()) || TOPIC.equals(properties.getExchangeType());
```

Comment:
To follow your pattern (that is cool) you can create a method called `isATopicExchange` and change here for something like this:
```
private boolean isAValidExchange(TunedRabbitProperties properties) {
        return isADirectExchange(properties) || isATopicExchange(properties));
}
```

BTW, nice job and thank you very much for your contribution!

Reply:
good idea

Reply:
done....